pkg/kubelet/allocation/allocation_manager.go

```diff
@@ -214,6 +232,37 @@ func (m *manager) Run(ctx context.Context) {
 	}()
 }
 
+// Gernerate pod resize completed event
+func (m *manager) emitPodResizeCompletionEvent(resizedPod *v1.Pod) string {
+	podResizeSource := &podResourceSummary{
+		InitContainers: make([]containerAllocation, 0),
+		Containers:     make([]containerAllocation, 0),
+	}
+
+	podutil.VisitContainers(&resizedPod.Spec, podutil.InitContainers|podutil.Containers,
+		func(allocatedContainer *v1.Container, containerType podutil.ContainerType) bool {
+			allocatedResources, exists := m.GetContainerResourceAllocation(resizedPod.UID, allocatedContainer.Name)
```

Comment:
The pod is already using the allocated values here, so you can just use the container resources directly (`allocatedResources := container.Resources`). In fact, this is better, since the resize we're processing here is using a snapshot of the allocated resources, but it's possible that a new resize was admitted in parallel.